### DISCUSSION TITLE
Możliwość ukrycia kolumny Częstotliwość

### DISCUSSION THREAD

**Discussion by NightPurrer** [TomaszGasior/RadioLista-v3#153] [2. Pomysły Ideas]
Obecnie możliwość ukrycia częstotliwości nie jest udostępniona, przycisk jest wyszarzony (Chyba tak to się nazywa) i nie da się w niego kliknąć.
Opcja ta nie była by zapewne praktycznie używana, ale w bardzo nietypowych, wybranych przypadkach by się przydała.

**Comment by TomaszGasior** [comment]
Hej, miło mi cię tu spotkać. :)

W moim odczuciu wykaz bez kolumny „częstotliwość” lub „nazwa” jest pozbawiony sensu: głównym celem wykazów RadioListy jest przedstawianie listy stacji radiowych, a nazwa i częstotliwość są integralnymi składnikami każdej stacji, służą bowiem do jej identyfikacji. Bez tych kolumn integralność wykazu nie jest zachowana. Interfejs aplikacji nie powinien pozwalać na robienie rzeczy bez sensu. :)

Oczywiście istnieje szansa, że jakiegoś konkretnego przypadku po prostu nie znam i nie rozumiem. Podaj proszę przykłady takich zastosowań, w których ma to sens, żeby kolumny z częstotliwością nie było. Chętnie się z nimi zapoznam.

**Comment by NightPurrer** [comment]
Może być to przydatne w bardzo niewielu przypadkach. W sumie ciężko mi powiedzieć, ale zdziwiłem się lekko i dlatego wpadłem na pomysł napisania :).

**Reply by TomaszGasior** [reply]
Rozumiem twoją perspektywę, jednakże, tak jak napisałem, aplikacja nie powinna pozwalać na robienie rzeczy bez sensu. Jeżeli możliwość robienia rzeczy bez sensu jest mało przydatna bądź nieprzydatna — tym bardziej nie powinno się dać tego zrobić. Takie są dobre praktyki tworzenia interfejsów użytkownika w aplikacjach: użytkownik powinien mieć możliwość konfigurowania tego, co ma sens, żeby było zmieniane. W moim odczuciu częstotliwość stacji zawsze powinna być widoczna. 

Oczywiście bardzo chętnie zapoznam się z kontrargumentami — po to jest forum dyskusyjne, żeby dyskutować i rozmawiać. Natomiast musisz podać konkretne przykłady nietypowych zastosowań albo linki do wykazów, w której taka funkcja byłaby potrzebna, żebym mógł lepiej zrozumieć problem, jaki chcemy wspólnie rozwiązać. Zrobienie takiej funkcji tak po prostu, żeby sobie była, nie ma sensu.

**Comment by NightPurrer** [comment]
Chciałem zrobić wykaz z linkami do strumieni stacji radiowych podanych w komentarzach, lecz po czasie stwierdziłem, że było by to bez sensu :D.
